deployment/uberspace-ansible/hosts.example

```diff
@@ -0,0 +1,3 @@
+[servers]
+machine001.example.com ansible_user=hpotter url=hagrid.example.com ansible_ssh_private_key_file=~/.ssh/machine001_examplecom_rsa
+
```

Comment:
As I said: I think it is a wise idea to separate it into separate roles and not to publish the actual inventory and host files.

Reply:
Well I'm not sure in what roles to seperate into. Also, this is an example file, not the actual hosts.

Reply:
For example a general installation of hagrid and one that covers the special needs of uberspace. The later one, for example, might then be replaced by nginx + postgres roles in order to cover a bare metal (or VM) server if one wants to deploy on such a system.